### ISSUE TITLE
pre-train시 학습 데이터

### ISSUE THREAD

**Issue by NJiHyeon** [Beomi/KcBERT#19]
안녕하세요! 좋은 모델과 코드를 열어주셔서 감사합니다.
다름이 아니라 제가 https://beomi.github.io/2021/03/15/KcBERT-MLM-Finetune/ 
이 사이트에 나와있는데로 추가 학습을 했었는데 
제 도메인에 맞는 데이터 [mask] 예측을 잘 하지 못하는 것 같아서,
vocab.txt를 제 학습 데이터로 만들어서 변경후 추가 학습을 하려하는데 이렇게 해도 괜찮을까요 ?
